src/Footer/ActiveCustomer/ActiveCustomer.jsx

```diff
@@ -1,14 +1,14 @@
-
-
-function ActiveCustomer() {
+import "./ActiveCustomer.css";
+export const ActiveCustomer = ({ arrayToRender }) => {
+  const activeCustomes = arrayToRender.filter(x => { if (x.status === "active") { return x.status } })
```

Comment:
You clearly still don't understand how filters works.

Reply:
i did map method than i changed it with filter one and forgot to delete the condition